constants/ws.py

```diff
@@ -29,6 +34,10 @@ class WSEvent:
     FILE_SAVE = "FILE_SAVE"
 
 
+ROOM_KEY = "room-{key}"  # session key to store room names to remember what rooms the user enterred.
```

Comment:
Replace `KEY` with `TYPE`, because this variable is used to group several rooms.